components/hi_mama.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "hi_mama",
+  "owner": ["@jcgoette"],
+  "manifest": "https://raw.githubusercontent.com/jcgoette/hi_mama_homeassistant/main/custom_components/hi_mama/manifest.json",
```

Comment:
`beautifulsoup4` is also required by the `scrape` integration, you should pin it with `>=` instead of `==` to ensure it will not break in the future.

Reply:
Thanks for pointing that out!

Would it make more sense to pin like this?

`"requirements": ["beautifulsoup4>=4.9.3,<5.0.0"]`

Reply:
That is probably safer.